src/xnnpack/internal.h

```diff
@@ -476,7 +476,7 @@ enum xnn_status xnn_reshape_batch_matrix_multiply_nc_qdu8_f32_qc8w(
 
 enum xnn_status xnn_setup_batch_matrix_multiply_nc_qdu8_f32_qc8w(
     xnn_operator_t batch_matrix_multiply_op, void* workspace,
-    const int8_t* input_a, const int8_t* input_b,
+    const int8_t* input_a, const float* input_b,
```

Comment:
input_b is byte, not float.
qdu8_f32_qc8w
qdu8 is input_a is uint8.  but apparently declared as int8.
f32 is output is float
qc8w is weights are int8